Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -113,15 +147,17 @@ class FakespotViewController: UIViewController, Themeable, Notifiable, UIAdaptiv
         setupView()
         listenForThemeChange(view)
         sendTelemetryOnAppear()
+
+        Task {
```

Comment:
What's the reason for us needing this Task here? We should definitely not do this if the user is not onboarded yet and also we should only fetch the analysis status if needsAnalysis is true.

Reply:
A check is indeed needed for opt in and needsAnalysis